Jenkinsfile

```diff
@@ -852,6 +852,25 @@ pipeline {
                         }
                     }
                 } // stage('Functional on CentOS 7')
+                stage('Functional on CentOS 7 with Valgrind') {
+                    when {
+                        beforeAgent true
+                        expression { ! skipStage() }
+                    }
+                    agent {
+                        label 'ci_vm9'
```

Comment:
I think you might want to use label params.CI_FUNCTIONAL_VM9_LABEL

Reply:
2c0c67ef355a6f60ffdb12fbc2506bffd3f27cf4